server/routes/referrals/completionDeadlineForm.test.ts

```diff
@@ -0,0 +1,126 @@
+import { Request } from 'express'
+import CompletionDeadlineForm from './completionDeadlineForm'
+
+describe('CompletionDeadlineForm', () => {
+  describe('createForm', () => {
+    it('returns a form and sanitizes the request body', async () => {
+      const req = {
+        body: {
+          'completion-deadline-year': '2021',
+          'completion-deadline-month': '09',
+          'completion-deadline-day': '12',
+        },
+      } as Request
+
+      await CompletionDeadlineForm.createForm(req)
+
+      expect(req.body['completion-deadline-year']).toBe(2021)
+      expect(req.body['completion-deadline-month']).toBe(9)
+      expect(req.body['completion-deadline-day']).toBe(12)
+    })
+
+    describe('with invalid data', () => {
+      it('does not modify the invalid fields', async () => {
```

Comment:
not a blocker - would be useful to know why (it does not modify invliad fields) for future readers.

Reply:
i guess this becomes immediately clear as you read the following test case.